src/libs/path.ts

```diff
@@ -0,0 +1,42 @@
+import { existsSync } from 'fs';
+import path from 'path';
+import protobuf from 'protobufjs';
+
+/**
+ * oarse the import paths from the import string
+ */
+export function getImportPaths(importString: string = '') {
+  return (importString?.split(',') || [])
+    .map(importPath => protobuf.util.path.normalize(path.posix.join(process.cwd(), importPath)));
+};
+
+/**
+ * override the resolvePath function of protobufjs.Root
+ * include the import paths to resolve the proto file
+ */
+export function getResolveProtoPathsFunction(imports: string[] = []) {
+  return (origin: string, target: string) => {
+    var normOrigin = protobuf.util.path.normalize(origin),
+      normTarget = protobuf.util.path.normalize(target);
+
+    var resolved = protobuf.util.path.resolve(normOrigin, normTarget, true);
+    var idx = resolved.lastIndexOf("google/protobuf/");
```

Comment:
Hard code is not recommended, maybe consider to use a constant instead?

Reply:
This is the source code copied from `protobuf.js`, and there will be no possibility of reuse here, so it should be fine to use literals?

FYI： https://github.com/protobufjs/protobuf.js/blob/547afa26f76e22e5463a17aec082b0b60cd951d8/cli/pbjs.js#L189